invenio_app_ils/documents/loaders/jsonschemas/document.py

```diff
@@ -267,19 +267,29 @@ def dump_extensions(self, obj):
 
         :params obj: content of the object's 'extensions' field
         """
-        ExtensionSchema = current_app.extensions[
-            "invenio-app-ils"
-        ].document_metadata_extensions.to_schema()
+        try:
+            extensions_config = getattr(
+                current_app.extensions["invenio-app-ils"],
+                "document_metadata_extensions",
+            )
+        except AttributeError:
+            return
```

Comment:
Is it ok to return `None`? Does that work? Or maybe it is expected to return empty dict?